cmd/vic-machine/create/create.go

```diff
@@ -818,28 +814,35 @@ func (c *Create) processNetwork(network *data.NetworkConfig, netName, pgName, st
 		}
 	}(network)
 
-	network.Destinations, network.Gateway, err = parseGatewaySpec(gateway)
-	if err != nil {
-		return fmt.Errorf("Invalid %s network gateway: %s", netName, err)
-	}
-	ipAddr, ipNet, err := net.ParseCIDR(staticIP)
-	if err != nil {
-		return fmt.Errorf("Provided %s network IP address %s has a wrong format", netName, staticIP)
+	if gateway != "" {
+		network.Destinations, network.Gateway, err = parseGatewaySpec(gateway)
+		if err != nil {
+			return fmt.Errorf("Invalid %s network gateway: %s", netName, err)
+		}
 	}
 
-	network.IP.IP = ipAddr
-	network.IP.Mask = ipNet.Mask
+	if staticIP != "" {
```

Comment:
https://github.com/vmware/vic/pull/3562/commits/606ab2080689216f0e1bc83dbb87a3a3ff364e2b#diff-5a99f2371e79e7b3ef78eefdc3419581R807
if gateway is not empty, staticIP is always not empty. :)